app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_counter
+
+  private
+
+  def update_counter
+    author.increment!(:comments_counter)
```

Comment:
- [ ] In order to follow a better flow, I think it would be better to have `post.increment!(:comments_counter)` instead of `author.increment!(:comments_counter)`. So please, work on this issue.